example.groovy

```diff
@@ -0,0 +1,5 @@
+class Example {
```

Comment:
⚠️ **[npm-groovy-lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class `Example` is the only class in `example.groovy`. In such a case the file and the class should have the same name.  ClassNameSameAsFilename